policyengine_canada/variables/gov/provinces/qc/tax/income/credits/qc_basic_personal_amount.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_canada.model_api import *
+
+
+class qc_basic_personal_amount(Variable):
+    value_type = float
+    entity = Person
+    label = "Quebec basic personal amount"
+    unit = CAD
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+    reference = (
+        "https://www.revenuquebec.ca/documents/en/formulaires/tp/2022-12/TP-1.D-V%282022-12%29.pdf",
+        "https://www.revenuquebec.ca/en/citizens/income-tax-return/completing-your-income-tax-return/how-to-complete-your-income-tax-return/line-by-line-help/350-to-398-1-non-refundable-tax-credits/line-350/",
+    )
+
+    def formula(household, period, parameters):
```

Comment:
need to check for is_head

Reply:
Also added a test file for `is_head`, let me know if no needed